CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### [Unreleased]
 Added
+- Added a.fast.klaviyo.com to img-src csp whitelist
```

Comment:
```suggestion
- Added a.fast.klaviyo.com to img-src policy csp whitelist
```

Reply:
this feels redundant to me, since the P in CSP stands for Policy already